kubernetes/config.properties

```diff
@@ -0,0 +1,8 @@
+inference_address=http://0.0.0.0:8080
+management_address=http://0.0.0.0:8081
+NUM_WORKERS=1
+number_of_gpu=1
```

Comment:
@dhanainme Does the number_of_gpu need to match with the n_gpu value in the TS helm chart?

Reply:
`n_gpu` would be exposed to TS container by docker. This should be set to `number_of_gpu` in `config.properties` above.

Have added a note